### PR TITLE
DOC: link to correct PR

### PR DESCRIPTION
**Pull request by erfannariman**
xref #39171, #18262, #35224

### CODE REVIEW

File: doc/source/whatsnew/v1.2.0.rst [hunk 1]
```diff
@@ -518,7 +518,7 @@ Deprecations
 - Deprecated parameter ``dtype`` of method :meth:`~Index.copy` for all :class:`Index` subclasses. Use the :meth:`~Index.astype` method instead for changing dtype (:issue:`35853`)
 - Deprecated parameters ``levels`` and ``codes`` in :meth:`MultiIndex.copy`. Use the :meth:`~MultiIndex.set_levels` and :meth:`~MultiIndex.set_codes` methods instead (:issue:`36685`)
 - Date parser functions :func:`~pandas.io.date_converters.parse_date_time`, :func:`~pandas.io.date_converters.parse_date_fields`, :func:`~pandas.io.date_converters.parse_all_fields` and :func:`~pandas.io.date_converters.generic_parser` from ``pandas.io.date_converters`` are deprecated and will be removed in a future version; use :func:`to_datetime` instead (:issue:`35741`)
-- :meth:`DataFrame.lookup` is deprecated and will be removed in a future version, use :meth:`DataFrame.melt` and :meth:`DataFrame.loc` instead (:issue:`18682`)
+- :meth:`DataFrame.lookup` is deprecated and will be removed in a future version, use :meth:`DataFrame.melt` and :meth:`DataFrame.loc` instead (:issue:`18262`)
```
Comment (jreback) [inline doc/source/whatsnew/v1.2.0.rst:521]: add 35224 (the link to the PR here) is actually more important but can have both

Comment (erfannariman) [reply doc/source/whatsnew/v1.2.0.rst]: Done

**Comment by jreback** [comment]
thanks @erfannariman

**Comment by jreback** [comment]
@meeseeksdev backport 1.2.x

**Pull request merged**